pillar/heroku/xpro.sls

```diff
@@ -147,20 +148,20 @@ heroku:
     MITXPRO_BASE_URL: {{ env_data.MITXPRO_BASE_URL }}
     MITXPRO_DB_CONN_MAX_AGE: 0
     MITXPRO_DB_DISABLE_SSL: True    # pgbouncer buildpack uses stunnel to handle encryption
-    MITXPRO_EMAIL_HOST: __vault__::secret-operations/global/mit-smtp>data>relay_host
-    MITXPRO_EMAIL_PASSWORD: __vault__::secret-operations/global/mit-smtp>data>relay_password
-    MITXPRO_EMAIL_PORT: 587
+    MITXPRO_EMAIL_HOST: {{ smtp_config.relay_host }}
+    MITXPRO_EMAIL_PASSWORD: {{ smtp_config.relay_password }}
+    MITXPRO_EMAIL_PORT: {{ smtp_config.relay_port }}
     MITXPRO_EMAIL_TLS: True
-    MITXPRO_EMAIL_USER: __vault__::secret-operations/global/mit-smtp>data>relay_username
+    MITXPRO_EMAIL_USER: {{ smtp_config.relay_username }}
     MITXPRO_ENVIRONMENT: {{ env_data.env_name }}
-    MITXPRO_FROM_EMAIL: 'MIT xPRO <xpro@mit.edu>'
+    MITXPRO_FROM_EMAIL: 'MIT xPRO <xpro@xpro.mit.edu>'
```

Comment:
Ugh, I'm going to have to check on this and get back to you. Do you think this is the "from" address that's used whenever a course team sends an email?

Reply:
Yeah I'm pretty sure it is - https://github.com/mitodl/mitxpro/blob/master/app.json#L318

Reply:
I don't think I'm going to get an answer today, and changing this again seems like it would be relatively minor. 

Let's go with this, so we can move ahead. 

```suggestion
    MITXPRO_FROM_EMAIL: 'MIT xPRO <support@xpro.mit.edu>'
```